pygeos/test/test_strtree.py

```diff
@@ -591,16 +597,238 @@ def test_query_touches_lines(line_tree, geometry, expected):
         # point within first polygon
         (pygeos.points(0, 0.5), []),
         # point is at edge of first polygon
-        (pygeos.points(HALF_UNIT_DIAG+EPS, 0), [0]),
+        (pygeos.points(HALF_UNIT_DIAG + EPS, 0), [0]),
         # box overlaps envelope of 2 polygons does not touch any at edge
         (box(0, 0, 1, 1), []),
         # box overlaps 2 polygons and touches edge of first
-        (box(HALF_UNIT_DIAG+EPS, 0, 2, 2), [0]),
+        (box(HALF_UNIT_DIAG + EPS, 0, 2, 2), [0]),
         # buffer overlaps 3 polygons but does not touch any at edge
-        (pygeos.buffer(pygeos.points(3, 3), HALF_UNIT_DIAG+EPS), []),
+        (pygeos.buffer(pygeos.points(3, 3), HALF_UNIT_DIAG + EPS), []),
         # only one point of multipoint within polygon but does not touch
-        (pygeos.multipoints([[0, 0], [7,7], [7, 8]]), []),
+        (pygeos.multipoints([[0, 0], [7, 7], [7, 8]]), []),
     ],
 )
 def test_query_touches_polygons(poly_tree, geometry, expected):
     assert_array_equal(poly_tree.query(geometry, predicate="touches"), expected)
+
+
+### Bulk query tests
+@pytest.mark.parametrize(
+    "geometry", [None, pygeos.points(0.5, 0.5), [[pygeos.points(0.5, 0.5)]]]
+)
+def test_query_bulk_wrong_dimemsions(tree, geometry):
+    with pytest.raises(TypeError) as ex:
+        tree.query_bulk(geometry)
+
+    assert str(ex.value) == "Array should be one dimensional"
+
+
+@pytest.mark.parametrize("geometry", [[], "foo", 1])
+def test_query_bulk_wrong_type(tree, geometry):
+    with pytest.raises(TypeError) as ex:
+        tree.query_bulk(geometry)
+
+    assert str(ex.value) == "Array should be of object dtype"
```

Comment:
For testing the message, you can add a `match="Array should be of object dtype"` to the pytest.raises call